examples/gno.land/p/grc/grc20/impl/impl.gno

```diff
@@ -50,36 +51,48 @@ func (t *Token) GetSymbol() string   { return t.symbol }
 func (t *Token) GetDecimals() uint   { return t.decimals }
 func (t *Token) TotalSupply() uint64 { return t.totalSupply }
 
-func (t *Token) BalanceOf(address std.Address) uint64 {
-	return t.balanceOf(address)
+func (t *Token) BalanceOf(owner string) uint64 {
```

Comment:
I still want to leave internal implementations as std.Address only, and force resolving only once at first. 
This way, public facing /r/ functions can choose to resolve, while grc20 isn't dependent on the name resolution strategy (and also is slightly faster). If we were to, I would enforce AddressOrName, (but I think that belongs outside of grc20 anyways)

Reply:
It's also easy to create wrappers for these methods, but if we support usernames, each wrapper would also have to do its own lookup. So the simpler previous option keeps things simpler.

Reply:
yep yep, that's what I planned to do, I just focused on cherry-picking your other PR and didn't change anything else.

no worries, I know how to do the good thing now :)

Reply:
Done